tests/app/unit/test_check.py

```diff
@@ -40,38 +40,37 @@ def github_client_error_raiser():
 @pytest.mark.parametrize(
     "repository_name, source_repository_name, expected_user_permission, expected_result",
     [
-        pytest.param("repo-1/name-1", "repo-1/name-1", "none", False, id="repo names match"),
         pytest.param(
             "repo-1/name-1",
             "user-1/name-1",
             "none",
-            True,
-            id="repo names don't match, o owner none permission",
+            False,
```

Comment:
Invert expected results - `check_fork_collaborator` now checks if the user is a collaborator - not the other way around (if the branch is external fork)